hw7/task1/stringBuilder.js

```diff
@@ -0,0 +1,21 @@
+const {Builder} = require('./builder');
+
+class StringBuilder extends Builder{
+    constructor(string){
+        super(string);
+    }
+    delete(index, count){
+        let str = this.string.substr(index,count);
+        this.string = this.string.replace(str, "");
```

Comment:
use `slice` method to extracting a section of a string and returns it as a new string, without modifying the original string.

Reply:
fixed - this method now locks how coppy() method rather then delete()